hardware_interface/include/hardware_interface/actuator.hpp

```diff
@@ -22,6 +22,7 @@
 #include "hardware_interface/actuator_interface.hpp"
 #include "hardware_interface/handle.hpp"
 #include "hardware_interface/hardware_info.hpp"
+#include "hardware_interface/helpers.hpp"
```

Comment:
2 things:
* the refactoring was a great idea but please let's use a little more specific filenames e.g. `lifecycle_helpers.hpp`
* the header include isn't required in this file but in the cpp like for the others ;)

Reply:
Done. Thanks for the review!